hugegraph-core/src/main/java/com/baidu/hugegraph/task/TaskManager.java

```diff
@@ -276,6 +288,11 @@ private void scheduleOrExecuteJob() {
             }
         } catch (Throwable e) {
             LOG.error("Exception occurred when schedule job", e);
+        } finally {
```

Comment:
only do executeTasksOnWorker and cancelTasksOnWorker when !server.master() or server.onlySingleNode(), and rename server to serverManager